docs/ref/django-admin.txt

```diff
@@ -984,6 +984,12 @@ By default, the SQL created is for running the migration in the forwards
 direction. Pass ``--backwards`` to generate the SQL for
 unapplying the migration instead.
 
+.. versionchanged:: 1.9
```

Comment:
I deliberately indented that note. I don't see it being important past 2.0 or 2.1. It's just regular behavior then. It doesn't break anything (unless the SQL shell somebody is using doesn't support comments :wink:). But keeping it around while people migrate might make sense.